scripting/get5/util.sp

```diff
@@ -260,15 +260,21 @@ stock void SetTeamInfo(int csTeam, const char[] name, const char[] flag = "",
 
   // Add Ready/Not ready tags to team name if in warmup.
   char taggedName[MAX_CVAR_LENGTH];
-  if ((g_GameState == Get5State_Warmup || g_GameState == Get5State_PreVeto) &&
-      !g_DoingBackupRestoreNow) {
-    MatchTeam matchTeam = CSTeamToMatchTeam(csTeam);
-    if (IsTeamReady(matchTeam)) {
-      Format(taggedName, sizeof(taggedName), "%T %s", "ReadyTag", LANG_SERVER, name);
+  if (g_ReadyTeamTag.BoolValue)
+  {
+    if ((g_GameState == Get5State_Warmup || g_GameState == Get5State_PreVeto) &&
+        !g_DoingBackupRestoreNow) {
+      MatchTeam matchTeam = CSTeamToMatchTeam(csTeam);
+      if (IsTeamReady(matchTeam)) {
+        Format(taggedName, sizeof(taggedName), "%T %s", "ReadyTag", LANG_SERVER, name);
+      } else {
+        Format(taggedName, sizeof(taggedName), "%T %s", "NotReadyTag", LANG_SERVER, name);
+      }
     } else {
-      Format(taggedName, sizeof(taggedName), "%T %s", "NotReadyTag", LANG_SERVER, name);
+      strcopy(taggedName, sizeof(taggedName), name);
```

Comment:
This if/else structure seems more confusing than it needs to be. Can you find a way to collapse the 
 strcopy(taggedName, sizeof(taggedName), name);` case (when the tags aren't added) so it's specified once? This code seems like it should only need 3 branches, not 4.

Reply:
Basically when it is set to 1, The [READY] and [NOT READY] tags need to go out when Warmup is over so in order to bring them to normal structure it was added.

Reply:
If we dont add the last else statement , when the Cvar is set to 0 the server wont even print team names and just print it as Counter Terrorist and Terrorist as the Team Names.